test/Stats.unittest.js

```diff
@@ -135,24 +167,18 @@ describe(
 					},
 					compiler: {
 						context: ""
+					},
+					hooks: {
+						stats: new SyncHook(["stats"])
 					}
 				});
 				const result = mockStats.toJson();
 				result.should.deepEqual({
-					assets: [],
-					assetsByChunkName: {},
-					children: [],
 					chunks: [],
-					entrypoints: {},
-					filteredAssets: 0,
-					filteredModules: 0,
 					errors: [],
-					hash: "1234",
-					modules: [],
+					warnings: [],
```

Comment:
This was the only test case that was changed, I think it is fine, because by default, we don't want to show anything to user if no hooks/plugins are applied, The only thing exported as JSON are errors, warnings, chunks, output Path, publicPath. (which are the default returen obj of toJson method). Should it be changed or should we export empty object rather than this? we can re-export the previous values too, but i don't think we should apply any defaults without hooks. Thoughts/Suggestions?

Reply:
Keep the old test results please, it let's us validate your end result.